.github/workflows/build.yaml

```diff
@@ -50,13 +50,77 @@ jobs:
           uv pip install -v --system --no-deps -e .
 
       - name: Run tests
-        env:
-          DAF_BUTLER_PLUGINS: ${{ github.workspace }}/python/lsst/dax/obscore/cli/resources.yaml
         run: |
           pytest -r a -v -n 3 --cov=lsst.dax.obscore --cov=tests --cov-report=xml --cov-report=term --cov-branch
+          butler obscore -h
 
       - name: Upload coverage to codecov
         uses: codecov/codecov-action@v4
         with:
           files: ./coverage.xml
           token: ${{ secrets.CODECOV_TOKEN }}
+
+  check-changes:
```

Comment:
@mwittgen is the plan to put this code in a reusable workflow? It looks like we only made this change in sphgeom and not all the other middleware packages.

Reply:
Yes.